### DISCUSSION TITLE
Cloudflare workers instead of aws lambdas?

### DISCUSSION THREAD

**Discussion by dionjwa** [nhost/nhost#1399] [Q&A]
How flexible is the `nhost` implementation of lambdas (cloud functions)?

I am going to be incorporating cloudflare workers (https://workers.cloudflare.com/) into my nhost stack, because they solve a large set of big problems for a low price, and include a great solution for blob storage. 

All the endpoints between e.g. hasura and the lambdas are publicly addressable, so could cloudflare workers be a substitute? The deployment story is simply git push->publish action -> done.

**Comment by elitan** [comment]
> How flexible is the nhost implementation of lambdas (cloud functions)?

Depends on what you mean by "flexible". Can you be more specific, please?

> All the endpoints between e.g. hasura and the lambdas are publicly addressable, so could cloudflare workers be a substitute? 

Yes, they are. See [System Environment Variabels](https://docs.nhost.io/platform/environment-variables#system-environment-variables).

**Comment by guicurcio** [comment]
You should be able you to use Cloudflare Workers as a substitute, since you can call any endpoints as you see fit (both in your front-end and in the Hasura console). You cannot currently change any settings in the Nhost dashboard that would allow you to deploy functions as workers to cloudflare instead of us creating aws lambdas (though I'm favor of Nhost cloud having cloudflare workers or deno rather than aws lambdas)
